itou/www/approvals_views/tests/test_detail.py

```diff
@@ -274,18 +275,20 @@ def test_prolongation_button(self, client):
         response = client.get(url)
         assertNotContains(response, reverse("approvals:declare_prolongation", kwargs={"approval_id": approval.id}))
 
+    @override_settings(TALLY_URL="https://tally.so")
     def test_edit_user_info_button(self, client):
         approval = ApprovalFactory(with_jobapplication=True)
         job_application = approval.jobapplication_set.get()
         siae_member = job_application.to_siae.members.first()
         client.force_login(siae_member)
+        url = reverse("approvals:detail", kwargs={"pk": approval.pk})
 
         user_info_edit_url = reverse(
-            "dashboard:edit_job_seeker_info", kwargs={"job_application_id": job_application.pk}
+            "dashboard:edit_job_seeker_info", kwargs={"job_seeker_pk": job_application.job_seeker.pk}
```

Comment:
Évite une requête (bon ok, on la fait plus bas, mais bon :)).
```suggestion
            "dashboard:edit_job_seeker_info", kwargs={"job_seeker_pk": job_application.job_seeker_id}
```